ci-operator/step-registry/idp/external-oidc/keycloak/aks/server/idp-external-oidc-keycloak-aks-server-commands.sh

```diff
@@ -0,0 +1,395 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+
+# Insall helm binary
+wget https://get.helm.sh/helm-v3.17.2-linux-amd64.tar.gz -O /tmp/helm.tar.gz
+tar -xvf /tmp/helm.tar.gz -C /tmp/
+chmod +x /tmp/linux-amd64/helm
+
+KEYCLOAK_ADMIN_TEST_USER="admin-$(< /dev/urandom tr -dc 'a-z0-9' | fold -w 6 | head -n 1 || true)"
+KEYCLOAK_ADMIN_TEST_PASSWORD="$(< /dev/urandom tr -dc 'a-z0-9' | fold -w 12 | head -n 1 || true)"
+KEYCLOAK_PREFIX="keycloak-$(< /dev/urandom tr -dc 'a-z0-9' | fold -w 9 | head -n 1 || true)"
+KEYCLOAK_HOST="$KEYCLOAK_PREFIX.$HYPERSHIFT_BASE_DOMAIN"
+CLUSTER_CONSOLE="$(oc whoami --show-console)"
```

Comment:
To avoid others and future updating guys being confused, how about adding a `#` comment to clarify something, like This is just a placeholder; If one hosted cluster to be created later will use this keycloak server as external oidc provider, this CLUSTER_CONSOLE is not right?